content/stability-working-group/index.md

```diff
@@ -0,0 +1,73 @@
+
+So we accept that not only will change happen, it must. This leads to how I would describe stability: the ability to react to change in an informed and graceful way. With change as a given, we must be able to react to it. But the other parts are also extremely important. Change can be extremely disruptive. In the ecosystem, change might mean:
+ - industrial users need to spend time away from their employer's goals
+ - library and tooling maintainers have to adapt and support multiple paths, taking away from feature development
+ - academic materials might need to be rewritten
+ - any other number of issues that are difficult to predict
+
+But we just said that change is going to happen! As such, _ecosystem stability_ is managing the *impact* of change as felt by all members described above. In order to reduce the negative impacts of change to the ecosystem there must be a graceful handling path, not a lack of change entirely. For example, deprecating something before removal. Another important factor here is the timing. Being predictable in change management means that the various groups can schedule work rather than being interrupted. This is why it is critical to be able to handle change as gracefully as possible.
+
+## The Stability Working Group
+
+Who is the SWG? Membership and meeting attendance is fluid, but we have representation from a cross section of the ecosystem. That includes people involved in or with background in language design, compiler, tooling and library maintenance, industrial usage, academia and consulting.
+
+Given that, we promote tools, procedures, and general culture to effectively manage change. In order to manage this we, as a community must be able to identify when change happens, plan for upcoming changes, and minimize the breadth and depth of the impact of any given change. Finally we aim to bring representation for the sections of the ecosystem that are generally less present.
+
+## What We Have Been Working On
+
+The SWG has worked on several projects, some of which continue to this day.
+
+We worked on a [review](https://jappie.me/analyzing-haskell-stability.html) of breakages tracked by the GHC team. This was illuminating to highlight and enumerate a high level set of changes. There is still room to do more extensive analysis here as well.
+
+Another project we have been working on is defining a framework for classifying GHC language extensions. This has been a long process. But ultimately is the type of communication enhancing improvement that is needed for predictability.
+
+More concretely we have defined a set of compiler features to help with stability. These were ideas that already existed in some form. Examples that we were able to bring to attention and have since landed in GHC include warning categories and deprecation of exports. Those both allow for more communication from compiler and library authors to users.
```

Comment:
Could you possibly link relevant proposals / GHC manual sections? It would be great to demonstrate as many tangible results as possible.